Tests/SwiftKueryORMTests/TestFind.swift

```diff
@@ -140,6 +141,51 @@ class TestFind: XCTestCase {
         })
     }
 
+    struct AgeFilter: QueryParams {
+        let age: [Int]
+    }
+
+    /**
+     Testing that the correct SQL Query is created to retrieve all the models.
+     Testing that correct amount of models are retrieved
+     */
+    func testFindAllMatchingArrayFilter() {
+        let connection: TestConnection = createConnection(.returnThreeRows)
+        Database.default = Database(single: connection)
+        let filterArray = [38,28,36]
+        let filter = AgeFilter(age: filterArray)
+        let numberOfFilters = filterArray.count
+        performTest(asyncTasks: { expectation in
+            Person.findAll(matching: filter) { array, error in
+                XCTAssertNil(error, "Find Failed: \(String(describing: error))")
+                XCTAssertNotNil(connection.query, "Find Failed: Query is nil")
+                if let query = connection.query {
+                    let expectedPrefix = "SELECT * FROM \"People\" WHERE"
+                    let expectedClauses = [["\"People\".\"age\" = ?1", "\"People\".\"age\" = ?2", "\"People\".\"age\" = ?3"]]
```

Comment:
Reading this makes me wonder why we aren't using a `WHERE IN` clause.  I'll raise an issue.